packages/stacks-classic/lib/components/notice/notice.less

```diff
@@ -157,41 +147,54 @@
         background-color: var(--_no-code-bg);
         color: var(--_no-code-fc);
         outline: var(--su-static1) solid var(--_no-code-bc);
-
-        border-radius: var(--br-md);
         padding-left: var(--su2);
         padding-right: var(--su2);
     }
 
-    & &--btn {
-        // TODO: decouple .s-notice--btn from .s-btn
+    button&--dismiss {
```

Comment:
Introduced `s-notice--dismiss` class here to make the dismiss button more explicit and style it accordingly.